src/server/eval/unary_operation_node.cpp

```diff
@@ -36,7 +36,7 @@ UnaryOperationNode::UnaryOperationNode(op::UnaryOperationEnum operation, const n
 string UnaryOperationNode::eval(GeoObject* obj) const
 {
 	// This is needed beacase of MSVC
-	assert(false);
```

Comment:
ok... maybe add the assert again and leave the return. and add a note that the function is not implemented and will never be called because the unary operation is not implemented in the parser.